From 6acd3dbaeaba5df3c91462932b6295fc107a7c7e Mon Sep 17 00:00:00 2001
Message-Id: <6acd3dbaeaba5df3c91462932b6295fc107a7c7e.1499608047.git.sleepingzucchini@gmail.com>
In-Reply-To: <cover.1499608047.git.sleepingzucchini@gmail.com>
References: <cover.1499608047.git.sleepingzucchini@gmail.com>
From: Tyler Olivieri <sleepingzucchini@gmail.com>
Date: Sat, 8 Jul 2017 20:21:00 -0400
Subject: [PATCH 4/5] staging: ccree: export symbol immediately following
 function

Patch to fix following checkpatch warning:
WARNING: EXPORT_SYMBOL(foo); should immediately follow its function/variable

Signed-off-by: Tyler Olivieri <sleepingzucchini@gmail.com>
---
 drivers/staging/ccree/ssi_fips.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/ccree/ssi_fips.c b/drivers/staging/ccree/ssi_fips.c
index fdc40f3..523057c 100644
--- a/drivers/staging/ccree/ssi_fips.c
+++ b/drivers/staging/ccree/ssi_fips.c
@@ -39,7 +39,6 @@ int ssi_fips_get_state(enum cc_fips_state_t *p_state)
 
 	return rc;
 }
-
 EXPORT_SYMBOL(ssi_fips_get_state);
 
 /*
@@ -57,5 +56,4 @@ int ssi_fips_get_error(enum cc_fips_error *p_err)
 
 	return rc;
 }
-
 EXPORT_SYMBOL(ssi_fips_get_error);
-- 
2.9.4

